vignettes/laminr.Rmd

```diff
@@ -149,29 +148,37 @@ DotPlot(seurat_obj, features = unique(markers$gene)) +
   ggplot2::theme(axis.text.x = ggplot2::element_text(angle = 90, vjust = 0.5))
 ```
 
-# Slice the tiledbsoma array store
+# Slice a TileDB-SOMA array store
 
-Alternatively to accessing individual CELLxGENE datasets from LaminDB, the **{cellxgene.census}** package can be used to slice the TileDB-SOMA array store for CELLxGENE Census, a concatenated version of most datasets in CELLxGENE.
+When artifacts contain TileDB-SOMA array stores they can be opened and sliced using the [**{tiledbsoma}** package](https://single-cell-data.github.io/TileDB-SOMA/index.html).
 
-```{r slice-tiledbsoma, eval=FALSE}
-library(cellxgene.census)
+```{r slice-tiledbsoma, eval = requireNamespace("tiledbsoma", quietly = TRUE)}
+# Set some environment variables to avoid an issue with {tiledbsoma}
+# https://github.com/chanzuckerberg/cellxgene-census/issues/1261
+Sys.setenv(TILEDB_VFS_S3_REGION = "us-west-2")
```

Comment:
aha, ok, the region is provided here, i see. But it is probably much better to provide via `tiledb` context.

Reply:
This was just a hacky workaround because there is a bug in the **{tiledbsoma}** package. I'll look into the context stuff but I'm not sure if we get all the necessary information from the API.

Reply:
I think that work-around is OK for now given you clearly marked it in the comments!

Sergei can help you find a cleaner solution over the next weeks & months.

I think this is good to merge!